insights/parsers/__init__.py

```diff
@@ -353,6 +350,81 @@ def parse_fixed_table(table_lines,
     return table_data
 
 
+def parse_delimited_table(table_lines,
```

Comment:
I think for backward compatibility purposes we should make an alias for this function (and mark it as deprecated when that becomes available)

Reply:
OK, will do.

Reply:
Actually, I'm not sure there's any point in that - if I've tidied up everything that used `parse_table`, why provide it again?

Reply:
The reason is that there are possibly other callers that you haven't fixed.

Reply:
@PaulWay this method must be explicitly added to the file `docs/api_index.rst` due to the way sphinx builds the parser catalog.